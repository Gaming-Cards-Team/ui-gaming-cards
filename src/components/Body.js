import React from "react";
import Banner from "./Banner";
import TopMessage from "./TopMessage";
import CardGallery from "./CardGallery";

const Body = () => {
  return (
    <div>
      <Banner></Banner>
      <TopMessage></TopMessage>
     
      <CardGallery></CardGallery>
    </div>
  );
};

export default Body;
