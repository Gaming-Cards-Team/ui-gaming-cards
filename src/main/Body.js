import React from 'react';

const Body = () => (
    <div id='body'>
       Body
    </div>
);

export default Body;