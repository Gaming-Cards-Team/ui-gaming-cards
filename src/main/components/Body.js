import React from "react";
import Banner from "./Banner";

const Body = () => {
  return (
      <Banner></Banner>
  );
};

export default Body;
