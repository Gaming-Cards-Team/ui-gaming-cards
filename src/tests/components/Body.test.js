import React from 'react';
import Body from '../../main/components/Body';

import renderer from 'react-test-renderer';

describe('Test Body component', () => {

    it('should validate the Body component', () => {
        
        const body = renderer.create(<Body />).toJSON();
        expect(body).toMatchSnapshot();
    });



});