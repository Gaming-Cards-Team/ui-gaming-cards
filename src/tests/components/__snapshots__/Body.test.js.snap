// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Body component should validate the Body component 1`] = `
<div
  data-testid="body"
>
  Body
</div>
`;
